src/Pact/Types/Term.hs

```diff
@@ -23,7 +23,7 @@
 
 module Pact.Types.Term
  (
-   Meta(..),mDocs,mMetas,
+   DocModel(..),dmDocs,dmModel,
```

Comment:
What would we call this type if we added a third thing to it (beyond doc and model)? Maybe we should keep the name `Meta`?

Reply:
You're right, we should standardize on calling it "meta" or "docmodel". I'll wait for Stu to weigh in.

Reply:
Yep, prefer "Meta"